docs/logs/configuring.asciidoc

```diff
@@ -3,95 +3,48 @@
 
 :ecs-link: {ecs-ref}[Elastic Common Schema (ECS)]
 
-== Configuring the Logs UI
+== Configuring the Logs data
 
-The `filebeat-*` index pattern is used to query data by default. If your logs
-are located in a different set of indices, use a different timestamp field, or
-contain parsed fields which you want to expose as individual columns, you can
-adjust the source configuration via the user interface or the {kib}
-configuration file.
+The default source configuration for logs is specified in the {kibana-ref}/logs-ui-settings-kb.html[Logs app settings] in the {kibana-ref}/settings.html[Kibana configuration file].
+The default configuration uses the `filebeat-*` index pattern to query the data.
+The default configuration also defines field settings for things like timestamps and container names, and the default columns to show in the logs pane.
 
-NOTE: Logs and Infrastructure share a common data source definition in
-each space. Changes in one of them can influence the data displayed in the
-other.
+If your logs have custom index patterns, or use non-default field settings, or contain parsed fields which you want to expose as individual columns, you can override the default settings.
+Click *Configuration* to change the settings.
+This opens the *Configure source* fly-out dialog.
```

Comment:
As of 7.4 the flyout will instead be a standalone tab called "Settings" (there's a PR at #42243 with a screenshot, not sure if that's handy at all). All of the settings themselves remain the same though, just a different location.

Reply:
Thanks. I was aware that was coming and it's on my list of things to follow up soon.